lib/gst/player/gstplayer.c

```diff
@@ -253,6 +270,9 @@ gst_player_finalize (GObject * object)
 
   GST_TRACE_OBJECT (self, "Finalizing");
 
+  if (self->priv->media_info)
+    g_object_unref (self->priv->media_info);
```

Comment:
The media_info should also always be destroyed when a the player is stopped / or a new URI is selected